scene/main/viewport.cpp

```diff
@@ -3598,7 +3593,7 @@ void Viewport::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "snap_2d_transforms_to_pixel"), "set_snap_2d_transforms_to_pixel", "is_snap_2d_transforms_to_pixel_enabled");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "snap_2d_vertices_to_pixel"), "set_snap_2d_vertices_to_pixel", "is_snap_2d_vertices_to_pixel_enabled");
 	ADD_GROUP("Rendering", "");
-	ADD_PROPERTY(PropertyInfo(Variant::INT, "msaa", PROPERTY_HINT_ENUM, String::utf8("Disabled (Fastest),2× (Average),4× (Slow),8× (Slowest)")), "set_msaa", "get_msaa");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "msaa", PROPERTY_HINT_ENUM, String::utf8("Disabled (Fastest),2� (Average),4� (Slow),8� (Slowest)")), "set_msaa", "get_msaa");
```

Comment:
Umm... what?

Reply:
Yeah that's part of why I was concerned about adding such exotic UTF-8 chars to otherwise ASCII code for a somewhat limited use case.

Reply:
I just fixed it by adding the characters manually. Looked up the alt code for it.